cmsmon-tools.spec

```diff
@@ -36,6 +43,7 @@ go get github.com/gizak/termui/v3
 # build monit tools
 pushd src/go/MONIT
   go build %flags monit.go
+  go build %flags ggus_parser.go
```

Comment:
use a for loop here to loop over monit_commands

Reply:
at the end of this spec, update the loop to use `%monit_command` and extra tools to create symlinks